packages/sdk/src/modules/loggerModule/wrapper.ts

```diff
@@ -0,0 +1,27 @@
+import type { LoggerInterface } from "./types";
+
+const methodsToExclude = ["log"];
+
+export function wrapLogger(
```

Comment:
We renamed it `injectMetadata` in middleware as it's more meaningful. Please adjust it here.

Reply:
Generally, it looks like that there:
```ts
import type { LoggerInterface } from "@vue-storefront/logger";

const METHODS_TO_SKIP = ["log"];

export function injectMetadata(
  logger: LoggerInterface,
  metadataGetter: (metadata: Record<string, any>) => Record<string, any>
): LoggerInterface {
  return new Proxy(logger, {
    get(target, prop) {
      const shouldSkipMethod =
        typeof target[prop] !== "function" ||
        METHODS_TO_SKIP.includes(prop as string);

      if (!shouldSkipMethod) {
        return (...args: any[]) => {
          const [message, metadata] = args;
          target[prop](message, {
            ...metadata,
            ...metadataGetter(metadata),
          });
        };
      }
      return target[prop];
    },
  });
}
```

Maybe we could copy paste it here at least for now? So same thing won't be different at least now.

Reply:
Changed to `injectMetadata` the rest is pretty much the same.